### PR TITLE
Optimize yolo adapter

### PR DESCRIPTION
**Pull request by akorobeinikov**

### CODE REVIEW

File: models/public/yolo-v4-tiny-tf/README.md [hunk 1]
```diff
@@ -17,11 +17,11 @@ YOLO v4 Tiny is a real-time object detection model based on ["YOLOv4: Optimal Sp
 
 Accuracy metrics obtained on [Common Objects in Context (COCO)](https://cocodataset.org/#home) validation dataset for converted model.
 
-| Metric                                                               | Value |
-| -------------------------------------------------------------------- | ------|
-| mAP                                                                  | 0.403 |
-| [COCO mAP (0.5)](http://cocodataset.org/#detection-eval)             | 0.463 |
-| [COCO mAP (0.5:0.05:0.95)](http://cocodataset.org/#detection-eval)   | 0.226 |
+| Metric                                                               | Value  |
```
Comment (vladimir-dudnik) [inline models/public/yolo-v4-tiny-tf/README.md:20]: why do you change representation format?

Comment (akorobeinikov) [reply models/public/yolo-v4-tiny-tf/README.md:20]: Other yolo models presented metrics as percentage

Comment (vladimir-dudnik) [reply models/public/yolo-v4-tiny-tf/README.md:20]: ok

**Review by vladimir-dudnik — APPROVED** [review]

**Comment by vladimir-dudnik** [comment]
validation detects no accuracy drops

**Review by eaidova — APPROVED** [review]

**Pull request merged**